packages/discord.js/src/managers/UserManager.js

```diff
@@ -102,7 +102,9 @@ class UserManager extends CachedManager {
    * @returns {Promise<Message>}
    */
   async send(user, options) {
-    return (await this.createDM(user)).send(options);
+    const dmChannel = await this.createDM(user);
+
+    return this.client.channels.createMessage(dmChannel, options);
   }
```

Comment:
Should this be removed, as it is now located on the more-appropriate channel manager? It only came into existence to allow sending messages to uncached users: https://github.com/discordjs/discord.js/pull/7087

This isn't an actual API route for users, so I would like to say removing this here makes sense. Thoughts, @vladfrangu?